docs/tools.md

```diff
@@ -58,7 +58,26 @@ There are few options available by CodeX Editor.
 | -- | -- | -- | -- |
 | `enableLineBreaks` | _Boolean_ | `false` | With this option, CodeX Editor won't handle Enter keydowns. Can be helpful for Tools like `<code>` where line breaks should be handled by default behaviour. |
 | `inlineToolbar` | _Boolean/Array_ | `false` | Pass `true` to enable the Inline Toolbar with all Tools, or pass an array with specified Tools list |
+| `onPase` | _Object_ | {} | Paste event handlers and configuration, see below.
 
 
+### Paste handling
+CodeX Editor handle paste on Blocks and provide API for plugins to process pasted data. When user paste content into Editor, pasted content is split into blocks.
+
+1. If plain text has been pasted, it is split by new line characters
+2. If HTML string has been pasted, it is split by block tags
+
+You should provide `onPaste` field in Tool configuration with following parameters:
+| Name | Type | Description |
+| -- | -- | -- | -- |
+| `handler` | `Function` | _Required_. Paste handler gets one argument `content`. `content` is HTML element extracted from pasted data. |
+| `tags` | `String[]` | _Optional_. If you provide tags parameter they will be extracted from pasted data and passed to your `handler` function. |
```

Comment:
List of tags that should be replaced by your Tool. Tag's `outerHTML` will be passed to the `handler` function